src/utils/oauth2_util.py

```diff
@@ -4,6 +4,16 @@
 import jwt
 
 
+roles_endpoint_access = {
+    "chef_routes.add": ["ADMINISTRATOR"],
```

Comment:
These ara not endpoints, this is not how you call endpoints in postman, please use endpoint urls such as `/chefs/<id>`.

Reply:
done.